web/src/admin/providers/saml/SAMLProviderFormForm.ts

```diff
@@ -186,8 +178,21 @@ export function renderForm({
                     help=${msg(
                         "Determines how authentik sends the response back to the Service Provider.",
                     )}
-                >
-                </ak-radio-input>
+                ></ak-radio-input>
+                <ak-text-input
+                    label=${msg("Issuer")}
+                    name="issuer"
+                    value="${provider.issuer || "authentik"}"
+                    required
+                    .errorMessages=${errors.issuer}
+                    help=${msg("Also known as EntityID.")}
```

Comment:
```suggestion
                    help=${msg("Also known as Entity ID.")}
```

From what I can find on the web, this seems to be the correct way of writing it